bootstrap/kubeadm/api/v1alpha3/kubeadmbootstrapconfig_types.go

```diff
@@ -68,6 +68,11 @@ type KubeadmConfigSpec struct {
 	// Format specifies the output format of the bootstrap data
 	// +optional
 	Format Format `json:"format,omitempty"`
+
+	// Verbosity is the number for the kubeadm log level verbosity.
+	// It overrides the `--v` flag in kubeadm commands.
+	// +optional
+	Verbosity *int `json:"verbosity,omitempty"`
```

Comment:
```suggestion
	Verbosity *int32 `json:"verbosity,omitempty"`
```
It looks like [klog implements this as an int32](https://github.com/kubernetes/klog/blob/12be8a0d907ae8c384359251205ba2099b1e85ed/klog.go#L207), so it might help to go ahead and match that type here.